packages/content-conformance/README.md

```diff
@@ -44,7 +44,7 @@ main()
 
 ## Configuration
 
-On its own, the content conformance system does not know where the files to be checked are located, or what rules should be used. To configure the checker, we create a `content-conformance.config.js` file located in your project's root directory.
+On its own, the content conformance system does not know where the files to be checked are located, or what rules should be used. To configure the checker, we create a `content-conformance.config.mjs` file located in your project's root directory.
```

Comment:
I've updated the config loading to look for `.mjs` instead, this ensures that it will work with node's native ESM support without having to specify `type: "module"` in the consumer's `package.json`